<!DOCTYPE html>

<html xmlns="http://www.w3.org/1999/xhtml">

<head>

<meta charset="utf-8" />
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<meta name="generator" content="pandoc" />
<meta http-equiv="X-UA-Compatible" content="IE=EDGE" />




<title>GI2019</title>

<script src="site_libs/jquery-1.11.3/jquery.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/cosmo.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<script src="site_libs/jqueryui-1.11.4/jquery-ui.min.js"></script>
<link href="site_libs/tocify-1.9.1/jquery.tocify.css" rel="stylesheet" />
<script src="site_libs/tocify-1.9.1/jquery.tocify.js"></script>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<link href="site_libs/highlightjs-9.12.0/textmate.css" rel="stylesheet" />
<script src="site_libs/highlightjs-9.12.0/highlight.js"></script>
<link href="site_libs/font-awesome-5.1.0/css/all.css" rel="stylesheet" />
<link href="site_libs/font-awesome-5.1.0/css/v4-shims.css" rel="stylesheet" />

<style type="text/css">code{white-space: pre;}</style>
<style type="text/css">
  pre:not([class]) {
    background-color: white;
  }
</style>
<script type="text/javascript">
if (window.hljs) {
  hljs.configure({languages: []});
  hljs.initHighlightingOnLoad();
  if (document.readyState && document.readyState === "complete") {
    window.setTimeout(function() { hljs.initHighlighting(); }, 0);
  }
}
</script>



<style type="text/css">
h1 {
  font-size: 34px;
}
h1.title {
  font-size: 38px;
}
h2 {
  font-size: 30px;
}
h3 {
  font-size: 24px;
}
h4 {
  font-size: 18px;
}
h5 {
  font-size: 16px;
}
h6 {
  font-size: 12px;
}
.table th:not([align]) {
  text-align: left;
}
</style>




<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
code {
  color: inherit;
  background-color: rgba(0, 0, 0, 0.04);
}
img {
  max-width:100%;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
summary {
  display: list-item;
}
</style>


<style type="text/css">
/* padding for bootstrap navbar */
body {
  padding-top: 51px;
  padding-bottom: 40px;
}
/* offset scroll position for anchor links (for fixed navbar)  */
.section h1 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h2 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h3 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h4 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h5 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h6 {
  padding-top: 56px;
  margin-top: -56px;
}
.dropdown-submenu {
  position: relative;
}
.dropdown-submenu>.dropdown-menu {
  top: 0;
  left: 100%;
  margin-top: -6px;
  margin-left: -1px;
  border-radius: 0 6px 6px 6px;
}
.dropdown-submenu:hover>.dropdown-menu {
  display: block;
}
.dropdown-submenu>a:after {
  display: block;
  content: " ";
  float: right;
  width: 0;
  height: 0;
  border-color: transparent;
  border-style: solid;
  border-width: 5px 0 5px 5px;
  border-left-color: #cccccc;
  margin-top: 5px;
  margin-right: -10px;
}
.dropdown-submenu:hover>a:after {
  border-left-color: #ffffff;
}
.dropdown-submenu.pull-left {
  float: none;
}
.dropdown-submenu.pull-left>.dropdown-menu {
  left: -100%;
  margin-left: 10px;
  border-radius: 6px 0 6px 6px;
}
</style>

<script>
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark it active
  menuAnchor.parent().addClass('active');

  // if it's got a parent navbar menu mark it active as well
  menuAnchor.closest('li.dropdown').addClass('active');
});
</script>

<!-- tabsets -->

<style type="text/css">
.tabset-dropdown > .nav-tabs {
  display: inline-table;
  max-height: 500px;
  min-height: 44px;
  overflow-y: auto;
  background: white;
  border: 1px solid #ddd;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs > li.active:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li.active:before {
  content: "&#xe258;";
  border: none;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs > li.active {
  display: block;
}

.tabset-dropdown > .nav-tabs > li > a,
.tabset-dropdown > .nav-tabs > li > a:focus,
.tabset-dropdown > .nav-tabs > li > a:hover {
  border: none;
  display: inline-block;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li {
  display: block;
  float: none;
}

.tabset-dropdown > .nav-tabs > li {
  display: none;
}
</style>

<!-- code folding -->



<style type="text/css">

#TOC {
  margin: 25px 0px 20px 0px;
}
@media (max-width: 768px) {
#TOC {
  position: relative;
  width: 100%;
}
}


.toc-content {
  padding-left: 30px;
  padding-right: 40px;
}

div.main-container {
  max-width: 1200px;
}

div.tocify {
  width: 20%;
  max-width: 260px;
  max-height: 85%;
}

@media (min-width: 768px) and (max-width: 991px) {
  div.tocify {
    width: 25%;
  }
}

@media (max-width: 767px) {
  div.tocify {
    width: 100%;
    max-width: none;
  }
}

.tocify ul, .tocify li {
  line-height: 20px;
}

.tocify-subheader .tocify-item {
  font-size: 0.90em;
}

.tocify .list-group-item {
  border-radius: 0px;
}


</style>



</head>

<body>


<div class="container-fluid main-container">


<!-- setup 3col/9col grid for toc_float and main content  -->
<div class="row-fluid">
<div class="col-xs-12 col-sm-4 col-md-3">
<div id="TOC" class="tocify">
</div>
</div>

<div class="toc-content col-xs-12 col-sm-8 col-md-9">




<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">lazappi's Twitter stats</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="index.html">Home</a>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        <li>
  <a href="about.html">About</a>
</li>
<li>
  <a href="https://github.com/lazappi/twitter-stats">
    <span class="fa fa-github fa-lg"></span>
     
  </a>
</li>
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div class="fluid-row" id="header">



<h1 class="title toc-ignore">GI2019</h1>
<h3 class="subtitle">Genome Informatics conference 2019</h3>
<h4 class="date">2019-11-09 11:07:02</h4>

</div>


<p><strong>Parameters</strong></p>
<table>
<thead>
<tr class="header">
<th align="left">Parameter</th>
<th align="left">Value</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left">hashtag</td>
<td align="left">#gi2019</td>
</tr>
<tr class="even">
<td align="left">start_day</td>
<td align="left">2019-11-06</td>
</tr>
<tr class="odd">
<td align="left">end_day</td>
<td align="left">2019-11-09</td>
</tr>
<tr class="even">
<td align="left">timezone</td>
<td align="left">America/New_York</td>
</tr>
<tr class="odd">
<td align="left">theme</td>
<td align="left">theme_light</td>
</tr>
<tr class="even">
<td align="left">accent</td>
<td align="left">#005190</td>
</tr>
<tr class="odd">
<td align="left">accent2</td>
<td align="left">#7FA8C7</td>
</tr>
<tr class="even">
<td align="left">kcore</td>
<td align="left">4</td>
</tr>
<tr class="odd">
<td align="left">topics_k</td>
<td align="left">6</td>
</tr>
<tr class="even">
<td align="left">bigram_filter</td>
<td align="left">5</td>
</tr>
<tr class="odd">
<td align="left">fixed</td>
<td align="left">FALSE</td>
</tr>
<tr class="even">
<td align="left">seed</td>
<td align="left">1</td>
</tr>
</tbody>
</table>
<div id="introduction" class="section level1">
<h1><span class="header-section-number">1</span> Introduction</h1>
<p>An analysis of tweets from the #gi2019 hashtag for the [Genome Informatics conference][GI2019], 6-9 November 2019, at Cold Spring Harbor Laboratories, New York, USA.</p>
<p>A total of 3157 tweets from 837 users were collected using the <code>rtweet</code> R package.</p>
</div>
<div id="timeline" class="section level1">
<h1><span class="header-section-number">2</span> Timeline</h1>
<div id="tweets-by-day" class="section level2">
<h2><span class="header-section-number">2.1</span> Tweets by day</h2>
<p><img src="figures/gi2019/tweets-by-day-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="tweets-by-day-and-time" class="section level2">
<h2><span class="header-section-number">2.2</span> Tweets by day and time</h2>
<p>Filtered for dates 2019-11-06 - 2019-11-09 in the America/New_York timezone.</p>
<p><img src="figures/gi2019/tweets-by-day-hour-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
</div>
<div id="users" class="section level1">
<h1><span class="header-section-number">3</span> Users</h1>
<div id="top-tweeters" class="section level2 tabset tabset-pills">
<h2><span class="header-section-number">3.1</span> Top tweeters</h2>
<div id="overall" class="section level3 unnumbered">
<h3>Overall</h3>
<p><img src="figures/gi2019/tweets-top-users-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="original" class="section level3 unnumbered">
<h3>Original</h3>
<p><img src="figures/gi2019/tweets-top-users-orig-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="retweets" class="section level3 unnumbered">
<h3>Retweets</h3>
<p><img src="figures/gi2019/tweets-top-users-retweet-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
</div>
<div id="retweet-proportion" class="section level2">
<h2><span class="header-section-number">3.2</span> Retweet proportion</h2>
<p><img src="figures/gi2019/tweets-retweet-prop-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="top-tweeters-timeline" class="section level2">
<h2><span class="header-section-number">3.3</span> Top tweeters timeline</h2>
<p><img src="figures/gi2019/tweets-top-users-time-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="top-tweeters-by-day" class="section level2 tabset tabset-pills">
<h2><span class="header-section-number">3.4</span> Top tweeters by day</h2>
<div id="overall-1" class="section level3 unnumbered tabset tabset-fade">
<h3>Overall</h3>
<div id="day-1" class="section level4 unnumbered">
<h4>Day 1</h4>
<p><img src="figures/gi2019/tweets-top-users-day1-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="day-2" class="section level4 unnumbered">
<h4>Day 2</h4>
<p><img src="figures/gi2019/tweets-top-users-day2-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="day-3" class="section level4 unnumbered">
<h4>Day 3</h4>
<p><img src="figures/gi2019/tweets-top-users-day3-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="day-4" class="section level4 unnumbered">
<h4>Day 4</h4>
<p><img src="figures/gi2019/tweets-top-users-day4-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
</div>
<div id="original-1" class="section level3 unnumbered tabset tabset-fade">
<h3>Original</h3>
<div id="day-1-1" class="section level4 unnumbered">
<h4>Day 1</h4>
<p><img src="figures/gi2019/tweets-top-users-orig-day1-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="day-2-1" class="section level4 unnumbered">
<h4>Day 2</h4>
<p><img src="figures/gi2019/tweets-top-users-orig-day2-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="day-3-1" class="section level4 unnumbered">
<h4>Day 3</h4>
<p><img src="figures/gi2019/tweets-top-users-orig-day3-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="day-4-1" class="section level4 unnumbered">
<h4>Day 4</h4>
<p><img src="figures/gi2019/tweets-top-users-orig-day4-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
</div>
<div id="retweets-1" class="section level3 unnumbered tabset tabset-fade">
<h3>Retweets</h3>
<div id="day-1-2" class="section level4 unnumbered">
<h4>Day 1</h4>
<p><img src="figures/gi2019/tweets-top-users-retweets-day1-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="day-2-2" class="section level4 unnumbered">
<h4>Day 2</h4>
<p><img src="figures/gi2019/tweets-top-users-retweets-day2-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="day-3-2" class="section level4 unnumbered">
<h4>Day 3</h4>
<p><img src="figures/gi2019/tweets-top-users-retweets-day3-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="day-4-2" class="section level4 unnumbered">
<h4>Day 4</h4>
<p><img src="figures/gi2019/tweets-top-users-retweets-day4-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
</div>
</div>
</div>
<div id="sources" class="section level1">
<h1><span class="header-section-number">4</span> Sources</h1>
<p><img src="figures/gi2019/tweets-top-sources-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="networks" class="section level1">
<h1><span class="header-section-number">5</span> Networks</h1>
<div id="replies" class="section level2">
<h2><span class="header-section-number">5.1</span> Replies</h2>
<p>The “replies network”, composed from users who reply directly to one another, coloured by PageRank.</p>
<p><img src="figures/gi2019/reply-network-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="mentions" class="section level2">
<h2><span class="header-section-number">5.2</span> Mentions</h2>
<p>The “mentions network”, where users mention other users in their tweets. Filtered for a k-core of 4. Node colour and size adjusted according to PageRank score.</p>
<p><img src="figures/gi2019/mentions-network-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
</div>
<div id="tweet-types" class="section level1">
<h1><span class="header-section-number">6</span> Tweet types</h1>
<div id="retweets-2" class="section level2 tabset tabset-pills">
<h2><span class="header-section-number">6.1</span> Retweets</h2>
<div id="proportion" class="section level3 unnumbered">
<h3>Proportion</h3>
<p><img src="figures/gi2019/is-retweet-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="count" class="section level3 unnumbered">
<h3>Count</h3>
<p><img src="figures/gi2019/retweet-count-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="top-10" class="section level3 unnumbered">
<h3>Top 10</h3>
<table>
<thead>
<tr>
<th style="text-align:left;">
screen_name
</th>
<th style="text-align:left;">
text
</th>
<th style="text-align:right;">
retweet_count
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
lh3lh3
</td>
<td style="text-align:left;">
I am looking for one or two postdocs working on bioinformatics and computational biology in general. Find me at #gi2019 if you are around. <a href="https://t.co/cZpnhzMAtr" class="uri">https://t.co/cZpnhzMAtr</a>
</td>
<td style="text-align:right;">
204
</td>
</tr>
<tr>
<td style="text-align:left;">
lpachter
</td>
<td style="text-align:left;">
I highly recommend “How to read (single-cell RNA-seq) PCA plots” (by <span class="citation">@vallens</span>): <a href="https://t.co/Co3A47SZTR" class="uri">https://t.co/Co3A47SZTR</a> #gi2019 He notes that “A particular danger [in interpreting T-shapes] is that it is tempting to interpret this as a bifurcation in the data.” (it almost never is). #gi2019
</td>
<td style="text-align:right;">
124
</td>
</tr>
<tr>
<td style="text-align:left;">
aphillippy
</td>
<td style="text-align:left;">
Heng is amazing. Go work for him. If he turns you down, come talk to me. I am also looking for students and postdocs at #gi2019 😆 <a href="https://t.co/uuuMq6gXLJ" class="uri">https://t.co/uuuMq6gXLJ</a>
</td>
<td style="text-align:right;">
72
</td>
</tr>
<tr>
<td style="text-align:left;">
hdashnow
</td>
<td style="text-align:left;">
I’m presenting STRling, our new method to detect novel (and reference) STR expansions from short-read data. <a href="https://t.co/hPyL7WBhkI" class="uri">https://t.co/hPyL7WBhkI</a> #gi2019 Big thanks to <span class="citation">@brent_p</span> who has done a huge amount of work on this algorithm and is an all around fantastic guy to work with.
</td>
<td style="text-align:right;">
42
</td>
</tr>
<tr>
<td style="text-align:left;">
BenLangmead
</td>
<td style="text-align:left;">
<p>Slides from my #gi2019 talk this morning, about work primarily by <span class="citation">@dnb_hopkins</span>: <a href="https://t.co/lZYnBK8dhm" class="uri">https://t.co/lZYnBK8dhm</a></p>
<p>Software: <a href="https://t.co/JALMMmfSca" class="uri">https://t.co/JALMMmfSca</a></p>
Library: <a href="https://t.co/8Yifw0NUsz" class="uri">https://t.co/8Yifw0NUsz</a>
</td>
<td style="text-align:right;">
28
</td>
</tr>
<tr>
<td style="text-align:left;">
deannachurch
</td>
<td style="text-align:left;">
Also- I’m looking for someone to lead my Informatics/analysis group- and single cell experience is a must! Help build the next generation of genomics tools using genome engineering! <a href="https://t.co/v8vzvssQUM" class="uri">https://t.co/v8vzvssQUM</a> #gi2019
</td>
<td style="text-align:right;">
27
</td>
</tr>
<tr>
<td style="text-align:left;">
StevenSalzberg1
</td>
<td style="text-align:left;">
centroFlye: finally the #bioinformatics community is starting to assemble centromeres, which have been a barrier to finishing the human genome for 25 years <span class="citation">@AndreyBzikadze</span> #gi2019
</td>
<td style="text-align:right;">
21
</td>
</tr>
<tr>
<td style="text-align:left;">
aaronquinlan
</td>
<td style="text-align:left;">
Tonight at #gi2019 <span class="citation">@hdashnow</span> will present STRling (pronounced “sterling”, the new approach she is developing with <span class="citation">@brent_p</span> to detect large STR expansions at both annotated and novel STR loci. I am so bummed to miss this fantastic meeting. <a href="https://t.co/6e4hw5qMkW" class="uri">https://t.co/6e4hw5qMkW</a>
</td>
<td style="text-align:right;">
20
</td>
</tr>
<tr>
<td style="text-align:left;">
GenomeInABottle
</td>
<td style="text-align:left;">
We’re excited for the first <span class="citation">@GenomeInABottle</span> benchmark for the MHC! See Justin Wagner’s poster at #gi2019 by <span class="citation">@infoecho</span> <span class="citation">@tobiasmarschal</span> <span class="citation">@MikkoRautiaine3</span> <span class="citation">@erikgarrison</span>… A Diploid Assembly-based Benchmark for Variants in the Major Histocompatibility Complex <a href="https://t.co/QegTBz6hsJ" class="uri">https://t.co/QegTBz6hsJ</a>
</td>
<td style="text-align:right;">
20
</td>
</tr>
<tr>
<td style="text-align:left;">
lpachter
</td>
<td style="text-align:left;">
A reminder: “cluster sizes in a t-SNE plot mean nothing” <a href="https://t.co/GUe2BTFLEO" class="uri">https://t.co/GUe2BTFLEO</a> #gi2019
</td>
<td style="text-align:right;">
19
</td>
</tr>
</tbody>
</table>
</div>
<div id="most-retweeted" class="section level3 unnumbered">
<h3>Most retweeted</h3>
<p><img src="figures/gi2019/most-retweeted-1.png" style="display: block; margin: auto;" /></p>
</div>
</div>
<div id="likes" class="section level2 tabset tabset-pills">
<h2><span class="header-section-number">6.2</span> Likes</h2>
<div id="proportion-1" class="section level3 unnumbered">
<h3>Proportion</h3>
<p><img src="figures/gi2019/has-favorite-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="count-1" class="section level3 unnumbered">
<h3>Count</h3>
<p><img src="figures/gi2019/favorite-count-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="top-10-1" class="section level3 unnumbered">
<h3>Top 10</h3>
<table>
<thead>
<tr>
<th style="text-align:left;">
screen_name
</th>
<th style="text-align:left;">
text
</th>
<th style="text-align:right;">
favorite_count
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
lpachter
</td>
<td style="text-align:left;">
I highly recommend “How to read (single-cell RNA-seq) PCA plots” (by <span class="citation">@vallens</span>): <a href="https://t.co/Co3A47SZTR" class="uri">https://t.co/Co3A47SZTR</a> #gi2019 He notes that “A particular danger [in interpreting T-shapes] is that it is tempting to interpret this as a bifurcation in the data.” (it almost never is). #gi2019
</td>
<td style="text-align:right;">
359
</td>
</tr>
<tr>
<td style="text-align:left;">
lh3lh3
</td>
<td style="text-align:left;">
I am looking for one or two postdocs working on bioinformatics and computational biology in general. Find me at #gi2019 if you are around. <a href="https://t.co/cZpnhzMAtr" class="uri">https://t.co/cZpnhzMAtr</a>
</td>
<td style="text-align:right;">
223
</td>
</tr>
<tr>
<td style="text-align:left;">
samstudio8
</td>
<td style="text-align:left;">
Made it to #gi2019 at <span class="citation">@CSHL</span>. My metagenomic assembly graph artwork is the book cover! <a href="https://t.co/T2X6EPqweC" class="uri">https://t.co/T2X6EPqweC</a>
</td>
<td style="text-align:right;">
182
</td>
</tr>
<tr>
<td style="text-align:left;">
aphillippy
</td>
<td style="text-align:left;">
Heng is amazing. Go work for him. If he turns you down, come talk to me. I am also looking for students and postdocs at #gi2019 😆 <a href="https://t.co/uuuMq6gXLJ" class="uri">https://t.co/uuuMq6gXLJ</a>
</td>
<td style="text-align:right;">
151
</td>
</tr>
<tr>
<td style="text-align:left;">
sexchrlab
</td>
<td style="text-align:left;">
<p>I can’t begin to tell you how much I love the openness of Genome Informatics. I know science has a long way to go, but the culture of sharing data and code, and emphasizing validation; it warms my heart.</p>
#GI2019
</td>
<td style="text-align:right;">
128
</td>
</tr>
<tr>
<td style="text-align:left;">
hdashnow
</td>
<td style="text-align:left;">
I’m presenting STRling, our new method to detect novel (and reference) STR expansions from short-read data. <a href="https://t.co/hPyL7WBhkI" class="uri">https://t.co/hPyL7WBhkI</a> #gi2019 Big thanks to <span class="citation">@brent_p</span> who has done a huge amount of work on this algorithm and is an all around fantastic guy to work with.
</td>
<td style="text-align:right;">
103
</td>
</tr>
<tr>
<td style="text-align:left;">
infoecho
</td>
<td style="text-align:left;">
A Manhattan plot for #GI2019 <a href="https://t.co/Jpamfgex8e" class="uri">https://t.co/Jpamfgex8e</a>
</td>
<td style="text-align:right;">
99
</td>
</tr>
<tr>
<td style="text-align:left;">
lpachter
</td>
<td style="text-align:left;">
A lightning <em>analysis</em> in progress by <span class="citation">@sinabooeshaghi</span> at #gi2019. He is running a live demo with kallisto | bustools: analysis of one million C. elegans cells in <em>1 minute</em> running a <span class="citation">@GoogleColab</span> notebook: <a href="https://t.co/MKE3W2ZjBi" class="uri">https://t.co/MKE3W2ZjBi</a> #gi2019 😮
</td>
<td style="text-align:right;">
90
</td>
</tr>
<tr>
<td style="text-align:left;">
chrisamiller
</td>
<td style="text-align:left;">
<p>SN: Not every bioinformatician gets to go to the algorithm ball. Some of them have to stay home and write pipelines</p>
Preach, brother 😆 #gi2019
</td>
<td style="text-align:right;">
89
</td>
</tr>
<tr>
<td style="text-align:left;">
BenLangmead
</td>
<td style="text-align:left;">
<p>Slides from my #gi2019 talk this morning, about work primarily by <span class="citation">@dnb_hopkins</span>: <a href="https://t.co/lZYnBK8dhm" class="uri">https://t.co/lZYnBK8dhm</a></p>
<p>Software: <a href="https://t.co/JALMMmfSca" class="uri">https://t.co/JALMMmfSca</a></p>
Library: <a href="https://t.co/8Yifw0NUsz" class="uri">https://t.co/8Yifw0NUsz</a>
</td>
<td style="text-align:right;">
82
</td>
</tr>
</tbody>
</table>
</div>
<div id="most-likes" class="section level3 unnumbered">
<h3>Most likes</h3>
<p><img src="figures/gi2019/most-favorited-1.png" style="display: block; margin: auto;" /></p>
</div>
</div>
<div id="quotes" class="section level2 tabset tabset-pills">
<h2><span class="header-section-number">6.3</span> Quotes</h2>
<div id="proportion-2" class="section level3 unnumbered">
<h3>Proportion</h3>
<p><img src="figures/gi2019/is-quote-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="count-2" class="section level3 unnumbered">
<h3>Count</h3>
<p><img src="figures/gi2019/quotes-count-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="top-10-2" class="section level3 unnumbered">
<h3>Top 10</h3>
<table>
<thead>
<tr>
<th style="text-align:left;">
screen_name
</th>
<th style="text-align:left;">
text
</th>
<th style="text-align:right;">
quote_count
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
AliciaOshlack
</td>
<td style="text-align:left;">
We already have 5 posters up on the website for you to look at before the poster session #GI2019 <a href="https://t.co/xrktvzLo6G" class="uri">https://t.co/xrktvzLo6G</a> <a href="https://t.co/qSNoUoImSe" class="uri">https://t.co/qSNoUoImSe</a>
</td>
<td style="text-align:right;">
4
</td>
</tr>
<tr>
<td style="text-align:left;">
AliciaOshlack
</td>
<td style="text-align:left;">
We already have two new posters on our <span class="citation">@F1000Research</span> Channel for this year’s conference #Gi2019. Check them out at your leisure <a href="https://t.co/Qi3BJYQCs5" class="uri">https://t.co/Qi3BJYQCs5</a>
</td>
<td style="text-align:right;">
4
</td>
</tr>
<tr>
<td style="text-align:left;">
ascendox
</td>
<td style="text-align:left;">
RT mike_schatz "RT AliciaOshlack: We already have two new posters on our F1000Research Channel for this year’s conference #Gi2019. Check them out at your leisure <a href="https://t.co/pz1UEPRaQ3%22" class="uri">https://t.co/pz1UEPRaQ3"</a>;
</td>
<td style="text-align:right;">
4
</td>
</tr>
<tr>
<td style="text-align:left;">
dsurujon
</td>
<td style="text-align:left;">
What a great idea! Here are my slides for my #gi2019 talk on Friday on how transcriptomic chaos can predict bacterial fitness <a href="https://t.co/zXVnqfAcfx" class="uri">https://t.co/zXVnqfAcfx</a> <a href="https://t.co/4DBC2pID7I" class="uri">https://t.co/4DBC2pID7I</a>
</td>
<td style="text-align:right;">
4
</td>
</tr>
<tr>
<td style="text-align:left;">
AdamJOrr
</td>
<td style="text-align:left;">
Guest tweeter in <span class="citation">@lh3lh3</span> ’s #gi2019 talk <a href="https://t.co/oLWLyS6ucx" class="uri">https://t.co/oLWLyS6ucx</a>
</td>
<td style="text-align:right;">
2
</td>
</tr>
<tr>
<td style="text-align:left;">
nephantes
</td>
<td style="text-align:left;">
Heng Li <span class="citation">@lh3lh3</span>: nextflow pipeline used for liftover hg38 -&amp;gt; hg19 ☑️ sucessfully. :) <span class="citation">@nextflowio</span> #gi2019 <a href="https://t.co/zdDOrcO8zo" class="uri">https://t.co/zdDOrcO8zo</a>
</td>
<td style="text-align:right;">
2
</td>
</tr>
<tr>
<td style="text-align:left;">
AliciaOshlack
</td>
<td style="text-align:left;">
I actually really like this photo. Who would have thought that <span class="citation">@pathogenomenick</span> and I only met for the first time 10 mintues before that #gi2019 <a href="https://t.co/DbTTm0vPsr" class="uri">https://t.co/DbTTm0vPsr</a>
</td>
<td style="text-align:right;">
2
</td>
</tr>
<tr>
<td style="text-align:left;">
AliciaOshlack
</td>
<td style="text-align:left;">
Sad that <span class="citation">@sexchrlab</span> wasn’t able to be here to do the opening of #gi2019 with us because of stupid flight cancellations <a href="https://t.co/DbTTm0vPsr" class="uri">https://t.co/DbTTm0vPsr</a>
</td>
<td style="text-align:right;">
2
</td>
</tr>
<tr>
<td style="text-align:left;">
aphillippy
</td>
<td style="text-align:left;">
Heng is amazing. Go work for him. If he turns you down, come talk to me. I am also looking for students and postdocs at #gi2019 😆 <a href="https://t.co/uuuMq6gXLJ" class="uri">https://t.co/uuuMq6gXLJ</a>
</td>
<td style="text-align:right;">
2
</td>
</tr>
<tr>
<td style="text-align:left;">
michelebusby
</td>
<td style="text-align:left;">
#gi2019 PhDs: Please don’t take yourself out the running for this because you’ve been using Heng’s tools for the last 5 years and are intimidated! I always learn a lot when I talk to Heng (as does the whole community) and this is a great opportunity. Take your shot. <a href="https://t.co/5grj3gBP1j" class="uri">https://t.co/5grj3gBP1j</a>
</td>
<td style="text-align:right;">
2
</td>
</tr>
</tbody>
</table>
</div>
<div id="most-quoted" class="section level3 unnumbered">
<h3>Most quoted</h3>
<p><img src="figures/gi2019/most-quoted-1.png" style="display: block; margin: auto;" /></p>
</div>
</div>
</div>
<div id="media" class="section level1 tabset tabset-pills">
<h1><span class="header-section-number">7</span> Media</h1>
<div id="proportion-3" class="section level2 unnumbered">
<h2>Proportion</h2>
<p><img src="figures/gi2019/has-media-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="top-10-3" class="section level2 unnumbered">
<h2>Top 10</h2>
<table>
<thead>
<tr>
<th style="text-align:left;">
screen_name
</th>
<th style="text-align:left;">
text
</th>
<th style="text-align:right;">
favorite_count
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
samstudio8
</td>
<td style="text-align:left;">
Made it to #gi2019 at <span class="citation">@CSHL</span>. My metagenomic assembly graph artwork is the book cover! <a href="https://t.co/T2X6EPqweC" class="uri">https://t.co/T2X6EPqweC</a>
</td>
<td style="text-align:right;">
182
</td>
</tr>
<tr>
<td style="text-align:left;">
infoecho
</td>
<td style="text-align:left;">
A Manhattan plot for #GI2019 <a href="https://t.co/Jpamfgex8e" class="uri">https://t.co/Jpamfgex8e</a>
</td>
<td style="text-align:right;">
99
</td>
</tr>
<tr>
<td style="text-align:left;">
lpachter
</td>
<td style="text-align:left;">
We’ve posted the kallisto | bustools poster presented by <span class="citation">@sinabooeshaghi</span> at #gi2019 on <span class="citation">@F1000Research</span> <a href="https://t.co/EVxAXZvGAC" class="uri">https://t.co/EVxAXZvGAC</a> <a href="https://t.co/inzS4JAeUZ" class="uri">https://t.co/inzS4JAeUZ</a>
</td>
<td style="text-align:right;">
69
</td>
</tr>
<tr>
<td style="text-align:left;">
aphillippy
</td>
<td style="text-align:left;">
<span class="citation">@AliciaOshlack</span> and <span class="citation">@pathogenomenick</span> kicking off #gi2019. So many legendary bioinformagicians in attendance. Love this meeting! <a href="https://t.co/BIMjUNC1E2" class="uri">https://t.co/BIMjUNC1E2</a>
</td>
<td style="text-align:right;">
64
</td>
</tr>
<tr>
<td style="text-align:left;">
infoecho
</td>
<td style="text-align:left;">
Real bioinformatics “file parsing”!!#gi2019 <a href="https://t.co/4vll3qI61u" class="uri">https://t.co/4vll3qI61u</a>
</td>
<td style="text-align:right;">
55
</td>
</tr>
<tr>
<td style="text-align:left;">
<strong>yoson</strong>
</td>
<td style="text-align:left;">
Guess which session this was (poll below) #gi2019 <a href="https://t.co/UBQJZdFLyn" class="uri">https://t.co/UBQJZdFLyn</a>
</td>
<td style="text-align:right;">
35
</td>
</tr>
<tr>
<td style="text-align:left;">
nimra236
</td>
<td style="text-align:left;">
Very interesting highlights of posters in “Lightining Talks” #gi2019 <a href="https://t.co/cRLMDdjiQB" class="uri">https://t.co/cRLMDdjiQB</a>
</td>
<td style="text-align:right;">
34
</td>
</tr>
<tr>
<td style="text-align:left;">
iddux
</td>
<td style="text-align:left;">
<span class="citation">@samstudio8</span> at #gi2019 on the difference between CPU and GPU tasks. “CPU is like two postdocs, they can do complex work but are easily distracted and need to do other things too. GPU is like an army of babies that can do just one simple thing, but can do it quickly and forever” <a href="https://t.co/rjQMzm7Tv3" class="uri">https://t.co/rjQMzm7Tv3</a>
</td>
<td style="text-align:right;">
34
</td>
</tr>
<tr>
<td style="text-align:left;">
ArangRhie
</td>
<td style="text-align:left;">
The genome informatics reunion :) <span class="citation">@sergekoren</span> <span class="citation">@AlexDilthey</span> <span class="citation">@aphillippy</span> <span class="citation">@traingene</span> <span class="citation">@chirgjain</span> <span class="citation">@sergeynurk</span> #gi2019 <a href="https://t.co/YMjWXtGUTb" class="uri">https://t.co/YMjWXtGUTb</a>
</td>
<td style="text-align:right;">
34
</td>
</tr>
<tr>
<td style="text-align:left;">
giuliaguidi
</td>
<td style="text-align:left;">
I was looking forward #GI2019 to catch up with friends and talk about our recent work on long-read overlap detection and alignment via sparse-matrix multiplication. Unforeseen circumstances are preventing me to attend —I know Twitter will help me to keep on top of things, though! <a href="https://t.co/BMyYt5rSJ4" class="uri">https://t.co/BMyYt5rSJ4</a>
</td>
<td style="text-align:right;">
32
</td>
</tr>
</tbody>
</table>
</div>
</div>
<div id="section" class="section level1 unnumbered toc-ignore">
<h1></h1>
<div id="most-liked-image" class="section level2">
<h2><span class="header-section-number">7.1</span> Most liked image</h2>
<p><img src="http://pbs.twimg.com/media/EIuW-SJW4AExEUJ.jpg" /></p>
</div>
</div>
<div id="tweet-text" class="section level1">
<h1><span class="header-section-number">8</span> Tweet text</h1>
<div id="word-cloud" class="section level2">
<h2><span class="header-section-number">8.1</span> Word cloud</h2>
<p>The top 100 words used 3 or more times.</p>
<p><img src="figures/gi2019/count-words-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="bigram-graph" class="section level2">
<h2><span class="header-section-number">8.2</span> Bigram graph</h2>
<p>Words that were tweeted next to each other at least 5 times.</p>
<p><img src="figures/gi2019/bigram-graph-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="topic-modelling" class="section level2">
<h2><span class="header-section-number">8.3</span> Topic modelling</h2>
<p>Top 10 words associated with 6 topics identified by LDA.</p>
<p><img src="figures/gi2019/topics-words-1.png" width="960" style="display: block; margin: auto;" /></p>
<div id="representative-tweets" class="section level3 tabset tabset-fade">
<h3><span class="header-section-number">8.3.1</span> Representative tweets</h3>
<p>Most representative tweets for each topic</p>
<div id="topic-1" class="section level4 unnumbered">
<h4>Topic 1</h4>
<table>
<thead>
<tr>
<th style="text-align:left;">
screen_name
</th>
<th style="text-align:left;">
text
</th>
<th style="text-align:right;">
gamma
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
Refasho
</td>
<td style="text-align:left;">
The fitting shoes #PixarSoul #gi2019 #WIZONELOVEIZONE #ThankfulThursday #BWFWarren #Obsession #ODMUnleashesGoons #JusticeForAsim #1026大切なユタの日 #10yearsofdanandphil #10월에_태어난_병아리_윈윈 #nyc #ny #344 #bloomfieldav #uppermontclair #montclair #newark #kearny #edison #byc <a href="https://t.co/efM3QsnsQH" class="uri">https://t.co/efM3QsnsQH</a>
</td>
<td style="text-align:right;">
0.9927014
</td>
</tr>
<tr>
<td style="text-align:left;">
bnlasse
</td>
<td style="text-align:left;">
<p>#gi2019 Harriet Dashnow, Exploring short tandem repeat (STR) expansions at known and novel loci in the human genome</p>
<p>STR=microsatellites; &amp;gt;40 STR disease loci STRling detect large STR expansions from short-read data</p>
👀👇 <a href="https://t.co/AnojqhXL3l" class="uri">https://t.co/AnojqhXL3l</a> (ref STR) <a href="https://t.co/lwvGd3K2iu" class="uri">https://t.co/lwvGd3K2iu</a>
</td>
<td style="text-align:right;">
0.9924431
</td>
</tr>
<tr>
<td style="text-align:left;">
Magdoll
</td>
<td style="text-align:left;">
Bzikadze: centroFlye for centromere assembly - identify centromeric reads, classify as prefix/internal/suffix, find rare/unique k-mers –&amp;gt; represent reads as k-mer clouds –&amp;gt; build contigs –&amp;gt; polish. Rub: need to deal w errors #gi2019
</td>
<td style="text-align:right;">
0.9921659
</td>
</tr>
<tr>
<td style="text-align:left;">
Magdoll
</td>
<td style="text-align:left;">
Eichler: Reference-free long read phased Hunan genome: start with HiFi (~33X) + Canu/wtbg2 —&amp;gt; SarrClust &amp;amp; StrandSeq (2.87X) —&amp;gt; DeepVariant to find SNPs —&amp;gt; WhatsHap &amp;amp; StrandSeq —&amp;gt; Peregrine or Canu to assemble haplotypes #gi2019
</td>
<td style="text-align:right;">
0.9921659
</td>
</tr>
<tr>
<td style="text-align:left;">
pmelsted
</td>
<td style="text-align:left;">
STR: several methods for reference STR (all using puns including the letters STR). Canvas (ataxia) disease, reference genome has alu element and AAAAG_n motif, disease elements have AAGGG_n motif, reference based methods fail to detect these #gi2019
</td>
<td style="text-align:right;">
0.9918675
</td>
</tr>
<tr>
<td style="text-align:left;">
bnlasse
</td>
<td style="text-align:left;">
<p>#gi2019 E Eichler, Long-read seq of structurally variant genomes <a href="https://t.co/T2tUbfURkD" class="uri">https://t.co/T2tUbfURkD</a> <a href="https://t.co/x3BzVvI1V4" class="uri">https://t.co/x3BzVvI1V4</a></p>
Accurate &amp;amp; rapid telomere-to-tel assembly of both haplotypes in diploid genome w/ &amp;gt;&amp;gt;accuracy &amp;amp; long(er) reads (HiFi + Strandseq) <a href="https://t.co/56gXyif3IO" class="uri">https://t.co/56gXyif3IO</a> &amp;amp; ++unpub 😉
</td>
<td style="text-align:right;">
0.9915455
</td>
</tr>
<tr>
<td style="text-align:left;">
lizworthey
</td>
<td style="text-align:left;">
KG: main goal of consortia is to provide real-time high definition reconstructions of the micro- (e.g., city/county) and macro-level (e.g., state/country) spread and evolution of WNV by sequencing from infected birds, mosquitoes, and patients #gi2019
</td>
<td style="text-align:right;">
0.9915455
</td>
</tr>
<tr>
<td style="text-align:left;">
pmelsted
</td>
<td style="text-align:left;">
SK: ONT real time, read-until can monitor the sequence, up to 512 channels at 450bp/sec. UNCALLED maps 10kbp/sec per thread, 75% of reads classified within 1 sec, [aside: ONT can eject reads from pores, saving on sequencing]. #gi2019
</td>
<td style="text-align:right;">
0.9911970
</td>
</tr>
<tr>
<td style="text-align:left;">
pmelsted
</td>
<td style="text-align:left;">
BL: Hyperlog (HLL) can process array of bits using SIMD values. Uses a bit more space by using a different base 1.19, fills up 8-bits rather than 6-bits with base 2. HLL performs better with lopsided data (large ratio of sizes between two sets) #gi2019
</td>
<td style="text-align:right;">
0.9908184
</td>
</tr>
<tr>
<td style="text-align:left;">
pathogenomenick
</td>
<td style="text-align:left;">
Andrey Bzikadze talking about centromere assembly with centroFlye: bigging up <span class="citation">@khmiga</span> and <span class="citation">@aphillippy</span>’s burgeoning T2T Consortium data which has huge haul of ultra-long reads and complementary technologies for human haploid genome: <a href="https://t.co/bNU6QpvEfI" class="uri">https://t.co/bNU6QpvEfI</a> #gi2019
</td>
<td style="text-align:right;">
0.9904059
</td>
</tr>
</tbody>
</table>
</div>
<div id="topic-2" class="section level4 unnumbered">
<h4>Topic 2</h4>
<table>
<thead>
<tr>
<th style="text-align:left;">
screen_name
</th>
<th style="text-align:left;">
text
</th>
<th style="text-align:right;">
gamma
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
pmelsted
</td>
<td style="text-align:left;">
HL: Incremental graph construction. Invariant property: linear path (original genome) and acyclic. Seq to graph alignment: Find seeds in segments, use linear chaining ignoring topology. Graph chaining, use graph topology to line up chains. Search best path with heuristics #gi2019
</td>
<td style="text-align:right;">
0.9927014
</td>
</tr>
<tr>
<td style="text-align:left;">
bnlasse
</td>
<td style="text-align:left;">
<p>#gi2019 Heng Li <span class="citation">@lh3lh3</span> The construct &amp;amp; utility of reference pan-genome graphs</p>
<p>Plan: start w/ GRCh38, incrementally add other genomes to construct graph, blacklist &amp;amp; decoy seqs for linear tools, updates preserve coordinate sys</p>
<a href="https://t.co/B6XVbWQX50" class="uri">https://t.co/B6XVbWQX50</a> <a href="https://t.co/NcTsAbyPyg" class="uri">https://t.co/NcTsAbyPyg</a>
</td>
<td style="text-align:right;">
0.9921659
</td>
</tr>
<tr>
<td style="text-align:left;">
bnlasse
</td>
<td style="text-align:left;">
<p>#gi2019 M Pertea, Efficient &amp;amp; robust transcriptome reconstruction from long-read RNA-seq alignments</p>
<p>Based on StringTie, short read transcriptome assembler: <a href="https://t.co/Kjg3ng0GPt" class="uri">https://t.co/Kjg3ng0GPt</a></p>
StringTie2, long read sln, overcome error rate and lower throughput 👀👇 <a href="https://t.co/00KzRLeRnH" class="uri">https://t.co/00KzRLeRnH</a>
</td>
<td style="text-align:right;">
0.9915455
</td>
</tr>
<tr>
<td style="text-align:left;">
pmelsted
</td>
<td style="text-align:left;">
HL: sad stories about Gnomad taking 6 years to move to Grch38 and someone running liftover to hg19 from Gnomad v3. New restricted format rGFA. Can incrementally add genomes to construct a graph. Updates preserve coordinate system #gi2019
</td>
<td style="text-align:right;">
0.9911970
</td>
</tr>
<tr>
<td style="text-align:left;">
Repealist_
</td>
<td style="text-align:left;">
<p>Incidental window seat right near the front, space to get work done,</p>
<p>and it starts raining just as I’m leaving Dublin?</p>
<p>All the promising makings of a great journey to #gi2019 and #SUNY.</p>
Slán a chairde 🇮🇪 tá sí ag dul go Nua-Eabhrac 🇺🇸! <a href="https://t.co/GTcyz95iNn" class="uri">https://t.co/GTcyz95iNn</a>
</td>
<td style="text-align:right;">
0.9908184
</td>
</tr>
<tr>
<td style="text-align:left;">
lpachter
</td>
<td style="text-align:left;">
I agree we should not hold scientific meetings on weekends. I also think we should not hold scientific meetings on weekdays. Scientists with kids face enormous logistical challenges on weekdays (kid pickup/dropoff, school events, activities, etc.) #gi2019 <a href="https://t.co/ifuJZuV3ci" class="uri">https://t.co/ifuJZuV3ci</a>
</td>
<td style="text-align:right;">
0.9904059
</td>
</tr>
<tr>
<td style="text-align:left;">
YiXing77
</td>
<td style="text-align:left;">
My student Yang Pan <span class="citation">@ypnngaa</span> will present his work on PEGASAS, pathway guided analysis of alternative splicing, at the upcoming Penn #RNA club next Tuesday! He also has a poster about PEGASAS at CSHL #gi2019 this week. #IOTN #Moonshot <a href="https://t.co/jzIJSD6wWU" class="uri">https://t.co/jzIJSD6wWU</a>
</td>
<td style="text-align:right;">
0.9904059
</td>
</tr>
<tr>
<td style="text-align:left;">
lpachter
</td>
<td style="text-align:left;">
The <span class="citation">@GoogleColab</span> notebook that <span class="citation">@sinabooeshaghi</span> just ran live at #gi2019 <a href="https://t.co/MKE3W2ZjBi" class="uri">https://t.co/MKE3W2ZjBi</a> Run it by just clicking on the link and selecting “Run all” from the “Runtime” menu. A minute to set up a machine from scratch and 1 min. for the demo taking reads -&amp;gt; PCA.
</td>
<td style="text-align:right;">
0.9904059
</td>
</tr>
<tr>
<td style="text-align:left;">
chrisamiller
</td>
<td style="text-align:left;">
<p>LINE1 activity correlates with poor immunotherapy response.</p>
Suggests that LINE1 high tumors are cloaked in a way that’s sidesteps activation with std imm. therapy (if they weren’t cloaked, they’d already be gone, since high LINE1 triggers response in normal cells!) #gi2019
</td>
<td style="text-align:right;">
0.9899545
</td>
</tr>
<tr>
<td style="text-align:left;">
FadelBerkadar
</td>
<td style="text-align:left;">
<p>A few words about novoSplice poster for the Genome Informatics meeting at CSHL #gi2019</p>
<ul>
<li>The poster explains the general workflow of novoSplice besides preliminary benchmarking results using both simulated and real reads. Algorithmic details of the splic…<a href="https://t.co/fxeHNScOal" class="uri">https://t.co/fxeHNScOal</a>
</td>
<td style="text-align:right;">
0.9899545
</td>
</tr>
</tbody>
</table></li>
</ul>
</div>
<div id="topic-3" class="section level4 unnumbered">
<h4>Topic 3</h4>
<table>
<thead>
<tr>
<th style="text-align:left;">
screen_name
</th>
<th style="text-align:left;">
text
</th>
<th style="text-align:right;">
gamma
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
bnlasse
</td>
<td style="text-align:left;">
<p>#gi2019 J Monlong: Genotyping structural vars in pangenome graphs using vg toolkit</p>
<p>SV catalogs: HGSVC/SVPOP/GIAB</p>
<p>vg toolkit-&amp;gt;integrate&amp;amp;genotype SVs; graphs from de novo assembly alignment performs better</p>
<a href="https://t.co/Pw6s925WAh" class="uri">https://t.co/Pw6s925WAh</a> <a href="https://t.co/ZrIR3V1BZh" class="uri">https://t.co/ZrIR3V1BZh</a> <a href="https://t.co/RL9gE55h2l" class="uri">https://t.co/RL9gE55h2l</a>
</td>
<td style="text-align:right;">
0.9924431
</td>
</tr>
<tr>
<td style="text-align:left;">
lizworthey
</td>
<td style="text-align:left;">
SLH: Process is RNA-seq, run data (on Dragen on AWS) through FusionMap, JAFFA, STARfusion, FusionCatcher, MapSplice, and TopHat fusion with overlap analysis, hierarchical prioritization, knowledge based filtering, and then CAP-CLIA validation of clinically relevant vars #gi2019
</td>
<td style="text-align:right;">
0.9924431
</td>
</tr>
<tr>
<td style="text-align:left;">
bnlasse
</td>
<td style="text-align:left;">
<p>#gi2019 S LaHaye: Utilization of ensemble approach for ID of driver fusions in peds cancer</p>
Ensemble approach reduces false +s 6 tools (FusionMap, JAFFA, STARfusion, FusionCatcher, MapSplice, TopHat Fusion) DRAGEN to speedup [seems to ID 98% in test] Seraseq as truth
</td>
<td style="text-align:right;">
0.9921659
</td>
</tr>
<tr>
<td style="text-align:left;">
pmelsted
</td>
<td style="text-align:left;">
TG: Filtering FPs using dual strand filter, nanopolish has highest sensitivity at high coverage. Phased variants in TP53 region using WhatsHap. Phasing of normal vs tumor tissue revealed cancer specific snps and a copy number difference between alleles in tumor #gi2019
</td>
<td style="text-align:right;">
0.9918675
</td>
</tr>
<tr>
<td style="text-align:left;">
chrisamiller
</td>
<td style="text-align:left;">
SL: Using a ensemble approach of 6 gene-fusion callers to identify fusions in ped cancer. Overlap (by at least 2 callers), prioritize based on evidence, and compare to a knowledge base. Final step is CAP/CLIA RT-PCR or sanger validation #gi2019
</td>
<td style="text-align:right;">
0.9918675
</td>
</tr>
<tr>
<td style="text-align:left;">
pmelsted
</td>
<td style="text-align:left;">
TG: High coverage of targeted regions means you can call SNPs better (except for homopolymer sites). Comp of tools, samtools, clair, medaka and nanopolish. 0.98 sensitivity for nanopolish and samtools at 200x cov (still a number of FPs), FPs often from a single strand #gi2019
</td>
<td style="text-align:right;">
0.9915455
</td>
</tr>
<tr>
<td style="text-align:left;">
bnlasse
</td>
<td style="text-align:left;">
<p>#gi2019 Medhat Mahmoud: PRINCESS—A framework for comprehensive detection and phasing of SNPs and structural variants (NGMLR/Minimap2 + Clair + Sniffles + WhatsHap + Princess-subtools + Nanopolish)</p>
<p>benchmark with GIAB; eg. resolving HLA-B</p>
<a href="https://t.co/OOcB5rMO1K" class="uri">https://t.co/OOcB5rMO1K</a>
</td>
<td style="text-align:right;">
0.9908184
</td>
</tr>
<tr>
<td style="text-align:left;">
chrisamiller
</td>
<td style="text-align:left;">
<p>DC: GRIDSS is unusual for making single-end breakpoint calls (nice for resolving CN events). Most single ends called when the other stretches into LINEs or PolyA tracts.</p>
Pipeline also does phasing from short reads (3-500bp) - works for a surprisingly large # of events #gi2019
</td>
<td style="text-align:right;">
0.9908184
</td>
</tr>
<tr>
<td style="text-align:left;">
lizworthey
</td>
<td style="text-align:left;">
JM: use overlap based approach to evaluate their genotyping; for del/inv require at least 50% coverage and 10% reciprocal overlap. For insertions req at least 50% of inserted sequence has to match nearby insertions otherwise considered error #gi2019
</td>
<td style="text-align:right;">
0.9899545
</td>
</tr>
<tr>
<td style="text-align:left;">
SeriousHorncat
</td>
<td style="text-align:left;">
Cheers at #gi2019 with <span class="citation">@lizworthey</span> and the rest of CGDS from UAB Medicine! Looking forward to showing a poster <a href="https://t.co/TiW2vt4Trt" class="uri">https://t.co/TiW2vt4Trt</a> on how we craft software to analyze molecular variation to investigate rare/undiagnosed disease to assist clinical decision making in the present <a href="https://t.co/lM1TlCDAzm" class="uri">https://t.co/lM1TlCDAzm</a>
</td>
<td style="text-align:right;">
0.9899545
</td>
</tr>
<tr>
<td style="text-align:left;">
SeriousHorncat
</td>
<td style="text-align:left;">
Cheers at #gi2019 with <span class="citation">@lizworthey</span> and the rest of CGDS from UAB Medicine! Looking forward to showing a poster <a href="https://t.co/TiW2vt4Trt" class="uri">https://t.co/TiW2vt4Trt</a> on how we craft software to analyze molecular variation to investigate rare/undiagnosed disease to assist clinical decision making in the present <a href="https://t.co/kLe3IHP4Rz" class="uri">https://t.co/kLe3IHP4Rz</a>
</td>
<td style="text-align:right;">
0.9899545
</td>
</tr>
</tbody>
</table>
</div>
<div id="topic-4" class="section level4 unnumbered">
<h4>Topic 4</h4>
<table>
<thead>
<tr>
<th style="text-align:left;">
screen_name
</th>
<th style="text-align:left;">
text
</th>
<th style="text-align:right;">
gamma
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
bnlasse
</td>
<td style="text-align:left;">
<p>#gi2019 A Balsubramani; Multi-resolution, interactive, atlas-scale integration of single-cell assays &amp;amp; exper.</p>
<p>Unify datasets from diff experiments? Connect w/ anchors, impute/co-embedding, refine on subpops and repeat.</p>
Their approach=Musica [I don’t think released quite yet]
</td>
<td style="text-align:right;">
0.9915455
</td>
</tr>
<tr>
<td style="text-align:left;">
bnlasse
</td>
<td style="text-align:left;">
<p>#gi2019 Xi Chen, ‘Tissue-specific enhancer functional networks for associating distal regulatory regions to disease’</p>
<p>How interpret/predict fxn of enhancers/disease-associated enhancers? Bayesian inference: <a href="https://t.co/JjuT1Ar8pq" class="uri">https://t.co/JjuT1Ar8pq</a> (try it!)</p>
autism e.g.: <a href="https://t.co/TlWmLl7CXY" class="uri">https://t.co/TlWmLl7CXY</a>
</td>
<td style="text-align:right;">
0.9911970
</td>
</tr>
<tr>
<td style="text-align:left;">
byuhobbes
</td>
<td style="text-align:left;">
<span class="citation">@jkpritch</span> <span class="citation">@jkpritch</span> Main challenges moving forward: How do we estimate the number of SNPs or genes that matter for a given trait? Can we use available data to infer core genes and trans-acting “master regulators”? Can we measure gene regulatory networks? #gi2019
</td>
<td style="text-align:right;">
0.9908184
</td>
</tr>
<tr>
<td style="text-align:left;">
byuhobbes
</td>
<td style="text-align:left;">
<span class="citation">@jkpritch</span> <span class="citation">@jkpritch</span> Recent metaanalysis of schizophrenia GWAS studies found 108 genome-wide significant loci; most hits are difficult to interpret, and only explain 10% of the observed heritability (“missing heritability” problem); enormous number of small effects #gi2019
</td>
<td style="text-align:right;">
0.9904059
</td>
</tr>
<tr>
<td style="text-align:left;">
lizworthey
</td>
<td style="text-align:left;">
gloria integration of ALzheimers disease genetics and myeloid cell genomics to revel novel risk mechanisms HiC, epigenomics, GWAS hits to identify causal genes and loci and validated in microglia #gi2019
</td>
<td style="text-align:right;">
0.9899545
</td>
</tr>
<tr>
<td style="text-align:left;">
BBFCreativeEnt
</td>
<td style="text-align:left;">
It must suck to support a racist president. This quote will bite you in the ass when you are up for reelection….It must suck to loose BIG to a democrat…and have to work in the private sector!#ThursdayMotivation #redcup #gi2019 #WashingtonPost #NYTimes #art #washingtinTimes <a href="https://t.co/aK9vZlnawo" class="uri">https://t.co/aK9vZlnawo</a>
</td>
<td style="text-align:right;">
0.9894586
</td>
</tr>
<tr>
<td style="text-align:left;">
Magdoll
</td>
<td style="text-align:left;">
.@dsurujon: bacterial fitness prediction essential in clinical setting for treatment outcome. need a generalizable model. generated lab strains of high/low fitness, RNA-seq found responding gene changes. #gi2019
</td>
<td style="text-align:right;">
0.9894586
</td>
</tr>
<tr>
<td style="text-align:left;">
lpachter
</td>
<td style="text-align:left;">
Now Akshay Balsubramani talking about integrating data from single-cell RNA-seq experiments (w/ <span class="citation">@anshulkundaje</span>). This is an important topic. Full disclosure: we’ve been partial to <span class="citation">@_romain_lopez_</span>’ et al.’s scVI in my lab but constant exciting new ideas in this area. #gi2019
</td>
<td style="text-align:right;">
0.9894586
</td>
</tr>
<tr>
<td style="text-align:left;">
Magdoll
</td>
<td style="text-align:left;">
Balsubramani: unifying single cell RNA-seq experiments. steps: anchor (find cells of similar cell types) –&amp;gt; impute features –&amp;gt; refine co-embedding. repeat. #gi2019
</td>
<td style="text-align:right;">
0.9894586
</td>
</tr>
<tr>
<td style="text-align:left;">
byuhobbes
</td>
<td style="text-align:left;">
<span class="citation">@_bakshay</span> <span class="citation">@_bakshay</span> unification strategy: connect corresponding cells of similar celltypes (anchors) across datasets, impute measured features within clusters in a joint graph, and refine co-embedding on cell subpopulations, repeating these steps #gi2019
</td>
<td style="text-align:right;">
0.9894586
</td>
</tr>
</tbody>
</table>
</div>
<div id="topic-5" class="section level4 unnumbered">
<h4>Topic 5</h4>
<table>
<thead>
<tr>
<th style="text-align:left;">
screen_name
</th>
<th style="text-align:left;">
text
</th>
<th style="text-align:right;">
gamma
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
bnlasse
</td>
<td style="text-align:left;">
<p>#gi2019 Naihui Zhou: Exploring 3D spatial dependency of gene expr using Markov random fields</p>
<p>hypothesis: expr is spatially dependent</p>
developed PHiMRF <a href="https://t.co/YQjn6CRXGO" class="uri">https://t.co/YQjn6CRXGO</a> prob model for RNA-seq data accounting for gene locals in 3D genome (spatial dependency for count data)
</td>
<td style="text-align:right;">
0.9933802
</td>
</tr>
<tr>
<td style="text-align:left;">
Magdoll
</td>
<td style="text-align:left;">
Kaminow: takes a haploid personal genome (ground truth), inserted all homozygous variants to ref, map RNA-seq using “STARconsensus”. Showed pan-human or super-population consensus genome did much better than the ref genome, as hoped. #gi2019
</td>
<td style="text-align:right;">
0.9911970
</td>
</tr>
<tr>
<td style="text-align:left;">
byuhobbes
</td>
<td style="text-align:left;">
BK: Recent approaches have investigated adding ALL TEH VARIANTS into a graph representation of the pangenome; alternative is a population-specific consensus genome; not a new idea, but bears re-investigation in context of contemporary functional (RNA-seq) data #gi2019
</td>
<td style="text-align:right;">
0.9911970
</td>
</tr>
<tr>
<td style="text-align:left;">
kasukawa
</td>
<td style="text-align:left;">
<p>Utilization of consensus genome, in which minor alleles are replaced to major ones.</p>
<p>Interestingly, pan-human consensus genome is adequate to reduce error rates. Even if using consensus genomes specific to subpopulations, the error rates were not reduced.</p>
#gi2019
</td>
<td style="text-align:right;">
0.9908184
</td>
</tr>
<tr>
<td style="text-align:left;">
bnlasse
</td>
<td style="text-align:left;">
<p>#gi2019 Karine Le Roch, ‘Comparative 3D genome organization in Apicomplexan parasites’</p>
<p>Overall, work supports link between genome organization and gene expression in more virulent pathogens</p>
learn more from their recent pubs: <a href="https://t.co/0bG5uj8tvD" class="uri">https://t.co/0bG5uj8tvD</a> <a href="https://t.co/Fc3Ar3U8re" class="uri">https://t.co/Fc3Ar3U8re</a>
</td>
<td style="text-align:right;">
0.9904059
</td>
</tr>
<tr>
<td style="text-align:left;">
Magdoll
</td>
<td style="text-align:left;">
Kaminow uses both 1000 genome (2548 genomes) + gnomAD (257k samples and growing) to create “consensus genome”, can be at the pan-human level, super-population level, and population level. I like this idea of having multiple layers of consensus. #gi2019
</td>
<td style="text-align:right;">
0.9904059
</td>
</tr>
<tr>
<td style="text-align:left;">
GenomeBiology
</td>
<td style="text-align:left;">
Jiao Sun, poster 176. Methods to explore the role of post-transcriptional regulation in cancer. Graph-based learning model on miRNA-mRNA bipartite graph to estimate protein expression #gi2019
</td>
<td style="text-align:right;">
0.9899545
</td>
</tr>
<tr>
<td style="text-align:left;">
Magdoll
</td>
<td style="text-align:left;">
Kaminow: mapping “error” (I prefer “ref mismatch”) decreases dramatically from ref to pan, but not much if using super-population level. It’s not clear why this is. My thought is this consensus idea, like the graph genome idea, is attractive but still in its infancy #gi2019
</td>
<td style="text-align:right;">
0.9894586
</td>
</tr>
<tr>
<td style="text-align:left;">
kasukawa
</td>
<td style="text-align:left;">
<p>quantifying isoform expression in scRNA-seq data with STARsolo-Quant #gi2019</p>
Quantifying each isoform based on the distribution function (by Maximum likelihood estimation) for distance to TTS and UMI barcoding.
</td>
<td style="text-align:right;">
0.9894586
</td>
</tr>
<tr>
<td style="text-align:left;">
Repealist_
</td>
<td style="text-align:left;">
<p>While we’re on break I’d just like to say a huge thank you to Catie and tthe team at <span class="citation">@CSHL</span> for organising such an informative #gi2019</p>
but also for fostering a great diversity of attendees - definitely encourage <span class="citation">@PrideinSTEM</span> <span class="citation">@LGBTSTEMDay</span> bioinformaticians to attend future meets!
</td>
<td style="text-align:right;">
0.9894586
</td>
</tr>
</tbody>
</table>
</div>
<div id="topic-6" class="section level4 unnumbered">
<h4>Topic 6</h4>
<table>
<thead>
<tr>
<th style="text-align:left;">
screen_name
</th>
<th style="text-align:left;">
text
</th>
<th style="text-align:right;">
gamma
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
bnlasse
</td>
<td style="text-align:left;">
<p>#gi2019 Sam Kovaka: Rapidly mapping raw <span class="citation">@nanopore</span> signal w/ UNCALLED to enable real-time targeted seq</p>
<p>ReadUntil seq bg: <a href="https://t.co/L5obZVjusq" class="uri">https://t.co/L5obZVjusq</a></p>
UNCALLED: Maps raw nanopore signals from fast5 files to large DNA refs (preprint soon!) <a href="https://t.co/gyahq5GDBd" class="uri">https://t.co/gyahq5GDBd</a>
</td>
<td style="text-align:right;">
0.9918675
</td>
</tr>
<tr>
<td style="text-align:left;">
sexchrlab
</td>
<td style="text-align:left;">
Genome Informatics has a culture of sharing code and data at the meeting. We are (afaik) the only mtg to routinely opt-out of tweeting/sharing, and we encourage poster sharing. That said, there is no explicit or implicit policy of releasing code/data at the meeting. #gi2019 <a href="https://t.co/ns0oisJkwX" class="uri">https://t.co/ns0oisJkwX</a>
</td>
<td style="text-align:right;">
0.9911970
</td>
</tr>
<tr>
<td style="text-align:left;">
pmelsted
</td>
<td style="text-align:left;">
Sam Kovaka: Rapidly mapping raw nanopore signal with UNCALLED for real time nanopore signal. Sequencing unwanted DNA, targeted sequence methods for nanopore PCR doesn’t work too well, limited to 5Kb. Crispr/cas9 can work up to 100kb, lowers throughput #gi2019
</td>
<td style="text-align:right;">
0.9911970
</td>
</tr>
<tr>
<td style="text-align:left;">
Magdoll
</td>
<td style="text-align:left;">
Dilthey: Applies MetMaps to PacBio HMP &amp;amp; ONT Zymo mock community achieves high read assignment on both. Caveat: reduced recall due to min RL 1kb; HMW DNA required. #gi2019 Paper here: <a href="https://t.co/KxePwF8jwM" class="uri">https://t.co/KxePwF8jwM</a>
</td>
<td style="text-align:right;">
0.9908184
</td>
</tr>
<tr>
<td style="text-align:left;">
pmelsted
</td>
<td style="text-align:left;">
SK: Raw signal is processed to event, k-mers are aligned to genone using FM-index with ambiguity, searches over k-mer space with probability thresholds (from the signal data). High FP in seed generation, uses clustering to find good seeds #gi2019
</td>
<td style="text-align:right;">
0.9908184
</td>
</tr>
<tr>
<td style="text-align:left;">
GenomeBiology
</td>
<td style="text-align:left;">
SH: was asked if she has compared RNA-seq data to DNA, to see if the fusions are due to trans-splicing or DNA translocations. Answer: good question. They’ve moved to using exome sequencing rather than whole-genome, which makes that harder to answer. #gi2019
</td>
<td style="text-align:right;">
0.9899545
</td>
</tr>
<tr>
<td style="text-align:left;">
lpachter
</td>
<td style="text-align:left;">
at #gi2019 RP is suggesting that transcriptome alignment (substrate for RSEM, eXpress, kallisto..) is not as good as genome alignment. A comparison was performed by <span class="citation">@alexlachmann</span> et al. in <a href="https://t.co/Bg5QhHA4EY" class="uri">https://t.co/Bg5QhHA4EY</a>. They conclude “the quality of the predictions is almost identical” <a href="https://t.co/M8XM0eKAer" class="uri">https://t.co/M8XM0eKAer</a>
</td>
<td style="text-align:right;">
0.9894586
</td>
</tr>
<tr>
<td style="text-align:left;">
byuhobbes
</td>
<td style="text-align:left;">
<span class="citation">@nomad421</span> <span class="citation">@nomad421</span> Randomly sampled 200 SRA datasets (109 after filtering; 69 bulk, 40 full-length single-cell); how do different parameter configurations impact abundance information? Evaluating on real data required some tedious investigation #gi2019
</td>
<td style="text-align:right;">
0.9894586
</td>
</tr>
<tr>
<td style="text-align:left;">
Repealist_
</td>
<td style="text-align:left;">
<p>Commuting to <span class="citation">@CSHL</span> #gi2019 meet and stopped off at Sutphin Blvd waiting for a train</p>
<p>Have already talked to three Bangla families and one Korean gal</p>
<p>And the morning fishmarkets are truly the view of dreams</p>
One for you <span class="citation">@MichaelsCoDub</span> <a href="https://t.co/glRsqxmruN" class="uri">https://t.co/glRsqxmruN</a>
</td>
<td style="text-align:right;">
0.9889112
</td>
</tr>
<tr>
<td style="text-align:left;">
cshlmeetings
</td>
<td style="text-align:left;">
<p>Starting tonight: Genome Informatics. Warm welcome to those on campus for #gi2019! For info: <a href="https://t.co/awkC8GZHTW" class="uri">https://t.co/awkC8GZHTW</a></p>
Please note: - Meeting’s opt-in social media policy - Friday’s morning session starts at 8:30 AM <a href="https://t.co/sLyAai23cu" class="uri">https://t.co/sLyAai23cu</a>
</td>
<td style="text-align:right;">
0.9889112
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
</div>
<div id="software" class="section level1">
<h1><span class="header-section-number">9</span> Software</h1>
<p>Software mentioned in Tweets with links to GitHub, BitBucket, Bioconductor or CRAN.</p>
<table>
<thead>
<tr class="header">
<th align="left">Name</th>
<th align="left">Type</th>
<th align="left">Link</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left">All-FIT</td>
<td align="left">GitHub</td>
<td align="left"><a href="https://github.com/khiabanianlab/all-fit" class="uri">https://github.com/khiabanianlab/all-fit</a></td>
</tr>
<tr class="even">
<td align="left">assembly</td>
<td align="left">GitHub</td>
<td align="left"><a href="https://github.com/hsnguyen/assembly" class="uri">https://github.com/hsnguyen/assembly</a></td>
</tr>
<tr class="odd">
<td align="left">bustools</td>
<td align="left">GitHub</td>
<td align="left"><a href="https://github.com/bustools/bustools" class="uri">https://github.com/bustools/bustools</a></td>
</tr>
<tr class="even">
<td align="left">centroFlye</td>
<td align="left">GitHub</td>
<td align="left"><a href="https://github.com/seryrzu/centroflye" class="uri">https://github.com/seryrzu/centroflye</a></td>
</tr>
<tr class="odd">
<td align="left">conterminator</td>
<td align="left">GitHub</td>
<td align="left"><a href="https://github.com/martin-steinegger/conterminator" class="uri">https://github.com/martin-steinegger/conterminator</a></td>
</tr>
<tr class="even">
<td align="left">dashing</td>
<td align="left">GitHub</td>
<td align="left"><a href="https://github.com/dnbaker/dashing" class="uri">https://github.com/dnbaker/dashing</a></td>
</tr>
<tr class="odd">
<td align="left">Flye</td>
<td align="left">GitHub</td>
<td align="left"><a href="https://github.com/fenderglass/flye" class="uri">https://github.com/fenderglass/flye</a></td>
</tr>
<tr class="even">
<td align="left">gfatools</td>
<td align="left">GitHub</td>
<td align="left"><a href="https://github.com/lh3/gfatools/blob/master/doc/rgfa.md" class="uri">https://github.com/lh3/gfatools/blob/master/doc/rgfa.md</a></td>
</tr>
<tr class="odd">
<td align="left">gridss</td>
<td align="left">GitHub</td>
<td align="left"><a href="https://github.com/papenfusslab/gridss" class="uri">https://github.com/papenfusslab/gridss</a></td>
</tr>
<tr class="even">
<td align="left">gridss-purple-linx</td>
<td align="left">GitHub</td>
<td align="left"><a href="https://github.com/hartwigmedical/gridss-purple-linx" class="uri">https://github.com/hartwigmedical/gridss-purple-linx</a></td>
</tr>
<tr class="odd">
<td align="left">GTEx_Brains</td>
<td align="left">GitHub</td>
<td align="left"><a href="https://github.com/sexchmlab/gtex_brains" class="uri">https://github.com/sexchmlab/gtex_brains</a></td>
</tr>
<tr class="even">
<td align="left">kallisto</td>
<td align="left">GitHub</td>
<td align="left"><a href="https://github.com/pachterlab/kallisto" class="uri">https://github.com/pachterlab/kallisto</a></td>
</tr>
<tr class="odd">
<td align="left">kb_python</td>
<td align="left">GitHub</td>
<td align="left"><a href="https://github.com/pachterlab/kb_python" class="uri">https://github.com/pachterlab/kb_python</a></td>
</tr>
<tr class="even">
<td align="left">lancet</td>
<td align="left">GitHub</td>
<td align="left"><a href="https://github.com/nygenome/lancet" class="uri">https://github.com/nygenome/lancet</a></td>
</tr>
<tr class="odd">
<td align="left">MetaMaps</td>
<td align="left">GitHub</td>
<td align="left"><a href="https://github.com/diltheylab/metamaps" class="uri">https://github.com/diltheylab/metamaps</a></td>
</tr>
<tr class="even">
<td align="left">minigraph</td>
<td align="left">GitHub</td>
<td align="left"><a href="https://github.com/lh3/minigraph" class="uri">https://github.com/lh3/minigraph</a></td>
</tr>
<tr class="odd">
<td align="left">mmseqs2</td>
<td align="left">GitHub</td>
<td align="left"><a href="https://github.com/soedinglab/mmseqs2" class="uri">https://github.com/soedinglab/mmseqs2</a></td>
</tr>
<tr class="even">
<td align="left">nanopore-methylation-utilities</td>
<td align="left">GitHub</td>
<td align="left"><a href="https://github.com/isaclee/nanopore-methylation-utilities" class="uri">https://github.com/isaclee/nanopore-methylation-utilities</a></td>
</tr>
<tr class="odd">
<td align="left">npBarcode</td>
<td align="left">GitHub</td>
<td align="left"><a href="https://github.com/hsnguyen/npbarcode" class="uri">https://github.com/hsnguyen/npbarcode</a></td>
</tr>
<tr class="even">
<td align="left">npScarf</td>
<td align="left">GitHub</td>
<td align="left"><a href="https://github.com/mdcao/npscarf" class="uri">https://github.com/mdcao/npscarf</a></td>
</tr>
<tr class="odd">
<td align="left">Palantir</td>
<td align="left">GitHub</td>
<td align="left"><a href="https://github.com/dpeerlab/palantir" class="uri">https://github.com/dpeerlab/palantir</a></td>
</tr>
<tr class="even">
<td align="left">phimrf</td>
<td align="left">GitHub</td>
<td align="left"><a href="https://github.com/ashleyzhou972/phimrf" class="uri">https://github.com/ashleyzhou972/phimrf</a></td>
</tr>
<tr class="odd">
<td align="left">PhiMRF</td>
<td align="left">GitHub</td>
<td align="left"><a href="https://github.com/ashleyzhou972/phimrf" class="uri">https://github.com/ashleyzhou972/phimrf</a></td>
</tr>
<tr class="even">
<td align="left">plass</td>
<td align="left">GitHub</td>
<td align="left"><a href="https://github.com/soedinglab/plass" class="uri">https://github.com/soedinglab/plass</a></td>
</tr>
<tr class="odd">
<td align="left">princess</td>
<td align="left">GitHub</td>
<td align="left"><a href="https://github.com/mehelmy/princess" class="uri">https://github.com/mehelmy/princess</a></td>
</tr>
<tr class="even">
<td align="left">pyranges</td>
<td align="left">GitHub</td>
<td align="left"><a href="https://github.com/biocore-ntnu/pyranges" class="uri">https://github.com/biocore-ntnu/pyranges</a></td>
</tr>
<tr class="odd">
<td align="left">readfq</td>
<td align="left">GitHub</td>
<td align="left"><a href="https://github.com/lh3/readfq" class="uri">https://github.com/lh3/readfq</a></td>
</tr>
<tr class="even">
<td align="left">RefineInsertions</td>
<td align="left">GitHub</td>
<td align="left"><a href="https://github.com/mkirsche/refineinsertions" class="uri">https://github.com/mkirsche/refineinsertions</a></td>
</tr>
<tr class="odd">
<td align="left">reticulatus</td>
<td align="left">GitHub</td>
<td align="left"><a href="https://github.com/samstudio8/reticulatus" class="uri">https://github.com/samstudio8/reticulatus</a></td>
</tr>
<tr class="even">
<td align="left">scVI</td>
<td align="left">GitHub</td>
<td align="left"><a href="https://github.com/yoseflab/scvi" class="uri">https://github.com/yoseflab/scvi</a></td>
</tr>
<tr class="odd">
<td align="left">SDA</td>
<td align="left">GitHub</td>
<td align="left"><a href="https://github.com/mvollger/sda" class="uri">https://github.com/mvollger/sda</a></td>
</tr>
<tr class="even">
<td align="left">ShinyOmics</td>
<td align="left">GitHub</td>
<td align="left"><a href="https://github.com/dsurujon/shinyomics" class="uri">https://github.com/dsurujon/shinyomics</a></td>
</tr>
<tr class="odd">
<td align="left">sketch</td>
<td align="left">GitHub</td>
<td align="left"><a href="https://github.com/dnbaker/sketch" class="uri">https://github.com/dnbaker/sketch</a></td>
</tr>
<tr class="even">
<td align="left">stringtie</td>
<td align="left">GitHub</td>
<td align="left"><a href="https://github.com/gpertea/stringtie" class="uri">https://github.com/gpertea/stringtie</a></td>
</tr>
<tr class="odd">
<td align="left">STRling</td>
<td align="left">GitHub</td>
<td align="left"><a href="https://github.com/quinlan-lab/strling" class="uri">https://github.com/quinlan-lab/strling</a></td>
</tr>
<tr class="even">
<td align="left">sv-genotyping-paper</td>
<td align="left">GitHub</td>
<td align="left"><a href="https://github.com/vgteam/sv-genotyping-paper" class="uri">https://github.com/vgteam/sv-genotyping-paper</a></td>
</tr>
<tr class="odd">
<td align="left">sveval</td>
<td align="left">GitHub</td>
<td align="left"><a href="https://github.com/jmonlong/sveval" class="uri">https://github.com/jmonlong/sveval</a></td>
</tr>
<tr class="even">
<td align="left">TALON</td>
<td align="left">GitHub</td>
<td align="left"><a href="https://github.com/dewyman/talon" class="uri">https://github.com/dewyman/talon</a></td>
</tr>
<tr class="odd">
<td align="left">TranscriptClean</td>
<td align="left">GitHub</td>
<td align="left"><a href="https://github.com/dewyman/transcriptclean" class="uri">https://github.com/dewyman/transcriptclean</a></td>
</tr>
<tr class="even">
<td align="left">UNCALLED</td>
<td align="left">GitHub</td>
<td align="left"><a href="https://github.com/skovaka/uncalled" class="uri">https://github.com/skovaka/uncalled</a></td>
</tr>
<tr class="odd">
<td align="left">vg</td>
<td align="left">GitHub</td>
<td align="left"><a href="https://github.com/vgteam/vg" class="uri">https://github.com/vgteam/vg</a></td>
</tr>
<tr class="even">
<td align="left">WHdenovo</td>
<td align="left">GitHub</td>
<td align="left"><a href="https://github.com/shilpagarg/whdenovo" class="uri">https://github.com/shilpagarg/whdenovo</a></td>
</tr>
<tr class="odd">
<td align="left">wnv-phylogeography</td>
<td align="left">GitHub</td>
<td align="left"><a href="https://github.com/andersen-lab/wnv-phylogeography" class="uri">https://github.com/andersen-lab/wnv-phylogeography</a></td>
</tr>
<tr class="even">
<td align="left">XPRESSpipe</td>
<td align="left">GitHub</td>
<td align="left"><a href="https://github.com/xpressyourself/xpresspipe" class="uri">https://github.com/xpressyourself/xpresspipe</a></td>
</tr>
</tbody>
</table>
</div>
<div id="session-info" class="section level1 unnumbered">
<h1>Session info</h1>
<pre><code>## R version 3.6.1 (2019-07-05)
## Platform: x86_64-w64-mingw32/x64 (64-bit)
## Running under: Windows 10 x64 (build 18362)
## 
## Matrix products: default
## 
## locale:
## [1] LC_COLLATE=English_Australia.1252  LC_CTYPE=English_Australia.1252   
## [3] LC_MONETARY=English_Australia.1252 LC_NUMERIC=C                      
## [5] LC_TIME=English_Australia.1252    
## 
## attached base packages:
## [1] stats     graphics  grDevices utils     datasets  methods   base     
## 
## other attached packages:
##  [1] fs_1.3.1           here_0.1           knitr_1.25        
##  [4] magick_2.2         webshot_0.5.1      viridis_0.5.1     
##  [7] viridisLite_0.3.0  wordcloud_2.6      RColorBrewer_1.1-2
## [10] ggraph_2.0.0       ggrepel_0.8.1      ggplot2_3.2.1     
## [13] topicmodels_0.2-8  tidytext_0.2.2     igraph_1.2.4.1    
## [16] stringr_1.4.0      purrr_0.3.3        forcats_0.4.0     
## [19] lubridate_1.7.4    tidyr_1.0.0        dplyr_0.8.3       
## [22] rtweet_0.6.9      
## 
## loaded via a namespace (and not attached):
##  [1] httr_1.4.1         tidygraph_1.1.2    jsonlite_1.6      
##  [4] assertthat_0.2.1   askpass_1.1        highr_0.8         
##  [7] stats4_3.6.1       yaml_2.2.0         progress_1.2.2    
## [10] slam_0.1-46        pillar_1.4.2       backports_1.1.5   
## [13] lattice_0.20-38    glue_1.3.1         digest_0.6.22     
## [16] polyclip_1.10-0    colorspace_1.4-1   plyr_1.8.4        
## [19] htmltools_0.4.0    Matrix_1.2-17      tm_0.7-6          
## [22] pkgconfig_2.0.3    scales_1.0.0       processx_3.4.1    
## [25] tweenr_1.0.1       ggforce_0.3.1      tibble_2.1.3      
## [28] openssl_1.4.1      generics_0.0.2     farver_1.1.0      
## [31] ellipsis_0.3.0     withr_2.1.2        lazyeval_0.2.2    
## [34] cli_1.1.0          NLP_0.2-0          magrittr_1.5      
## [37] crayon_1.3.4       ps_1.3.0           evaluate_0.14     
## [40] tokenizers_0.2.1   janeaustenr_0.1.5  fansi_0.4.0       
## [43] SnowballC_0.6.0    MASS_7.3-51.4      xml2_1.2.2        
## [46] tools_3.6.1        prettyunits_1.0.2  hms_0.5.2         
## [49] lifecycle_0.1.0    munsell_0.5.0      callr_3.3.2       
## [52] compiler_3.6.1     rlang_0.4.1        grid_3.6.1        
## [55] labeling_0.3       rmarkdown_1.16     gtable_0.3.0      
## [58] curl_4.2           reshape2_1.4.3     graphlayouts_0.5.0
## [61] R6_2.4.0           gridExtra_2.3      utf8_1.1.4        
## [64] zeallot_0.1.0      rprojroot_1.3-2    modeltools_0.2-22 
## [67] stringi_1.4.3      parallel_3.6.1     Rcpp_1.0.2        
## [70] vctrs_0.2.0        tidyselect_0.2.5   xfun_0.10</code></pre>
</div>



</div>
</div>

</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.header').parent('thead').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- tabsets -->

<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});

$(document).ready(function () {
  $('.tabset-dropdown > .nav-tabs > li').click(function () {
    $(this).parent().toggleClass('nav-tabs-open')
  });
});
</script>

<!-- code folding -->

<script>
$(document).ready(function ()  {

    // move toc-ignore selectors from section div to header
    $('div.section.toc-ignore')
        .removeClass('toc-ignore')
        .children('h1,h2,h3,h4,h5').addClass('toc-ignore');

    // establish options
    var options = {
      selectors: "h1,h2,h3",
      theme: "bootstrap3",
      context: '.toc-content',
      hashGenerator: function (text) {
        return text.replace(/[.\\/?&!#<>]/g, '').replace(/\s/g, '_').toLowerCase();
      },
      ignoreSelector: ".toc-ignore",
      scrollTo: 0
    };
    options.showAndHide = true;
    options.smoothScroll = true;

    // tocify
    var toc = $("#TOC").tocify(options).data("toc-tocify");
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
